data/operators/man_made/pipeline.json

```diff
@@ -2426,14 +2426,9 @@
       "displayName": "Kinder Morgan Texas Pipeline LLC",
       "id": "kindermorgantexaspipelinellc-0a386a",
       "locationSet": {"include": ["us"]},
-      "matchNames": [
-        "kinder morgan tejas pipeline",
-        "kinder morgan tejas pipeline llc"
-      ],
       "tags": {
         "man_made": "pipeline",
         "operator": "Kinder Morgan Texas Pipeline LLC",
-        "operator:wikidata": "Q110987354"
```

Comment:
Unless I’ve missed something, [Q110987354](https://www.wikidata.org/wiki/Q110987354) is the item for Kinder Morgan Texas Pipeline LLC. This line is consistent with NSI’s practice within the operators/ tree, even if you disagree with the use of `operator:wikidata` specifically on pipelines.

Reply:
re-added it.